ixwebsocket/IXWebSocketPerMessageDeflateOptions.cpp

```diff
@@ -85,11 +85,7 @@ namespace ix
 
             if (startsWith(token, "server_max_window_bits="))
             {
```

Comment:
stoi returns 0 on errors right ?

Reply:
No, it throws an exception (invalid arg for non number strings, or out of range for non convertible argument). Even if you handle exceptions locally and you have a "high" exceptional case ratio, you still remain an order of magnitude faster than the `stringstream` solution http://coliru.stacked-crooked.com/a/abc62fdec42443f5

I'll make a quick MR in case you want to review how such local handling would look like (I mean in case you don't handle exceptions further down the execution stack).